### PR TITLE
Added creation of backends in OpenDJ

### PR DESCRIPTION
**Pull request by dj-wasabi**
##### ISSUE TYPE

- New Module Pull Request
##### COMPONENT NAME

opendj_backend
##### ANSIBLE VERSION

```
2.2
```
##### SUMMARY

This module will create/delete backends in OpenDJ (LDAP directory server). This module will make use of the command 'dsconfig' and 'manage-tasks', which are installed with the installation of OpenDJ.

### CODE REVIEW

**Comment by dj-wasabi** [comment]
Can someone help with fixing issue with "Shippable" ?

**Comment by gregdek** [comment]
Thanks @dj-wasabi for this PR. This PR requires revisions, either because it fails to build or by reviewer request. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by dj-wasabi** [comment]
ready_for_review

**Comment by gregdek** [comment]
Thanks @dj-wasabi. To the current maintainers, @dj-wasabi please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

File: identity/opendj/opendj_backend.py [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+        description:
+            - The base DN. Example dc=example,dc=com.
+        required: True
+    type:
+        description:
+            - The type of backend you want to use for OpenDJ.
+        required: false
+        choices: ["je", "pdb"]
+        default: "je"
+    enable:
```
Comment (resmo) [inline identity/opendj/opendj_backend.py]: I would call this `enabled` as indicates a state not an action.

Comment (dj-wasabi) [reply identity/opendj/opendj_backend.py]: Yes, you are right @resmo . Renamed it.

File: identity/opendj/opendj_backend.py [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+        else:
+            return self._module.fail_json(msg="Error message while getting list backends: " + str(stderr))
+
+    def create_backend(self, opendj_bindir, hostname, port, password_method, name, base_dn, bind_dn, type, enabled):
+        my_command = [
+            opendj_bindir + '/dsconfig',
+            'create-backend',
+            '--backend-name', name,
+            '--set', 'base-dn:' + base_dn,
+            '--set', 'enabled:' + str(enable).lower(),
```
Comment (resmo) [inline identity/opendj/opendj_backend.py]: IMHO you missed one: `'--set', 'enabled:' + str(enabled).lower(),`?

Comment (dj-wasabi) [reply identity/opendj/opendj_backend.py]: oh, this is embarrassing. Yes, fixed I checked the file again to see if I missed more. luckily non other found. :-)

**Pull request closed without merging**